pkg/golinters/revive.go

```diff
@@ -305,6 +305,8 @@ var allRules = append([]lint.Rule{
 	&rule.NestedStructs{},
 	&rule.IfReturnRule{},
 	&rule.UselessBreak{},
+	&rule.OptimizeOperandsOrderRule{},
+	&rule.TimeEqualRule{},
```

Comment:
Can you also add a sample configuration to [`.golangci.example.yml`](https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml)?